codegen/templates/endpoint_register.tmpl

```diff
@@ -54,12 +57,15 @@ func Register(g *zanzibar.Gateway, router *zanzibar.Router) {
 						},
 				),
 				{{end -}}
-			}, endpoints.{{$endpoint.HandlerName}}.HandleRequest).Handle,
+			}, endpoints.{{$e.HandlerName}}.HandleRequest).Handle,
 			{{- else -}}
-			endpoints.{{$endpoint.HandlerName}}.HandleRequest,
+			endpoints.{{$e.HandlerName}}.HandleRequest,
 			{{- end}}
 			
 		),
 	)
+	{{else -}}
+	tchannelServer.Register("{{.Method.ThriftService}}", "{{.Method.Name}}", endpoints.{{.HandlerName}})
```

Comment:
Why is registering a tchannel endpoint so simple q_q.

We should refactor `router.Register()` to be `Register(endpointId, handleId, handler)` instead of `Register(*RouterEndpoint)`